case_specification.tex

```diff
@@ -0,0 +1,300 @@
+\subsection{Assumptions}
+
+\begin{frame}
+	\frametitle{Assumptions}
+	\begin{itemize}
+		\item fuel cycle facility parameters (throughput, availability)
```

Comment:
The fact that the simulations made assumptions about each of these things is obvious a priori. It is less obvious what those parameters, compositions, and material flow assumptions actually were.

Reply:
This frame was like an introduction to the many assumptions I was going to talk about..? If you feel like this slide is redundant I can get rid of it